keps/sig-cluster-lifecycle/0023-documentation-for-images.md

```diff
@@ -0,0 +1,148 @@
+
+
+# Documentation for images
+
+Open https://www.websequencediagrams.com/ and paste the spec for the desired image:
+
+- [kubeadm init](#kubeadm-init)
+- [kubeadm join (and join --master)](#kubeadm-join-and-join---master)
+- [kubeadm reset](#kubeadm-reset)
+- [kubeadm upgrade](#kubeadm-upgrade)
+- [kubeadm upgrade node](#kubeadm-upgrade-node)
+
+## kubeadm init
+
+```
+title kubeadm init (interactions with v1beta1 config)
+
+participant "user" as u 
+participant "kubeadm" as k
+participant "kubelet" as kk
+participant "node\n(api object)" as n
+participant "kubeadm-config\nConfigMap" as cm
+participant "kubeproxy-config\nConfigMap" as kpcm 
+participant "kubelet-config\nConfigMap-1.*" as kcm
+
+u->k:provide\nInitConfiguration (with NodeRegistrationOptions, Master)\nClusterConfiguration\nKubeProxy component config\nKubelet component config 
```

Comment:
`Kubelet` -> `kubelet`
(case-sensitive search-and-replace would solve as there are more than one instances.)

Reply:
`component config` -> `component configuration`?
(present in other places too)